compose-stability-analyzer-idea/src/main/kotlin/com/skydoves/compose/stability/idea/toolwindow/ComposableStabilityCollector.kt

```
@@ -0,0 +1,178 @@
+                packageName = packageName.ifEmpty { "<default>" },
+                fileName = fileName,
+                filePath = filePath,
+                line = line,
+                isSkippable = skippable,
+                isRestartable = restartable,
+                isRuntime = !skippable && restartable,
+                parameters = parameters,
+              ),
+            )
+          }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)